backend/data/src/main/kotlin/io/tolgee/service/slackIntegration/SavedSlackMessageService.kt

```diff
@@ -0,0 +1,73 @@
+package io.tolgee.service.slackIntegration
+
+import io.tolgee.model.slackIntegration.SavedSlackMessage
+import io.tolgee.repository.slackIntegration.SavedSlackMessageRepository
+import io.tolgee.repository.slackIntegration.SlackConfigRepository
+import org.springframework.scheduling.annotation.Scheduled
+import org.springframework.stereotype.Service
+import java.time.LocalDateTime
+import java.time.ZoneId
+import java.util.*
+
+@Service
+class SavedSlackMessageService(
+  private val savedSlackMessageRepository: SavedSlackMessageRepository,
+  private val slackConfigRepository: SlackConfigRepository,
+) {
+  fun create(savedSlackMessage: SavedSlackMessage): SavedSlackMessage {
```

Comment:
Usually, the service methods are @Transactional. Please add transactional to the service methods and remove it from controller methods if you don't have good reasons to keep it on controller level.